src/MessagingInput/MessagingInput.less

```diff
@@ -195,29 +196,6 @@
   .MessagingInput--TextField {
     .margin--right--none();
     min-height: 2rem;
-
-    .TextArea--input {
-      // Padding to account for the inside-textarea Send button in mobile UI.
-      //  Ensures that text is to the left of the Send button.
-      padding-right: 2.5rem;
-    }
-  }
-
-  .MessagingInput--ShowUploadAttachmentButton {
-    .TextArea--input {
-      padding-right: 5.1875rem; // 83px
-    }
-
-    .FileInput {
-      position: absolute;
-      right: 2.25rem;
-      bottom: 0;
-
-      .FileInput--Icon {
-        .margin--left--xs();
-        margin-right: 0.3125rem; // 5px
-      }
-    }
```

Comment:
These styles were all previously necessary to account for the space taken up by the in-input send button on mobile. With mobile now more closely mirroring desktop, fewer overrides are necessary for mobile! :tada:

_Before_

<img width="500" alt="before-3" src="https://user-images.githubusercontent.com/12616928/126706798-4f7a3763-1d83-479c-b8d4-eadb6e6cdfd8.png">

_After_

<img width="500" alt="after-12" src="https://user-images.githubusercontent.com/12616928/126706836-03e895ca-827f-4dc7-b3fb-41b1e64bf840.png">